builds/misc/templates/image-linux.yaml

```diff
@@ -3,21 +3,22 @@ parameters:
   imageName: ''
   namespace: 'microsoft'
   project: ''
+  version: ''
```

Comment:
The default value here should be $(Build.BuildNumber) or it will break the regular image build

Reply:
No it won't. The script has the logic to figure out if a string is empty or not. If the string is empty, it will use the `$(Build.BuildNumber)`. I made sure of it by code inspection + running the test to verify.

For Image build pipeline: 

https://dev.azure.com/msazure/One/_build/results?buildId=32358519&view=logs&j=cb472997-8ad7-5b8a-089b-78e3f1db5255

For Image release pipeline

https://msazure.visualstudio.com/One/_build/results?buildId=32358553&view=logs&j=7b5daa7d-adff-5805-f451-ad7193802469&t=efbf70ad-10b8-5ad3-3a7c-d6ad7449acae